sidebars.js

```diff
@@ -3,11 +3,7 @@ module.exports = {
     {
       type: 'category',
       label: 'Get Started',
-      items: [
-        'latest/tutorial/introduction',
-        'latest/tutorial/quick-start',
```

Comment:
We no longer have the quick start, so this had to go!

Reply:
@felixrieseberg can we add this doc to the sidebar as well under `Get Started`?